common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1139,6 +1139,10 @@ public final class LC {
     public static final KnownKey smtp_to_lmtp_enabled = KnownKey.newKey(false);
     public static final KnownKey smtp_to_lmtp_port = KnownKey.newKey(7024);
 
+    @Supported
+    public static final KnownKey modify_prefs_logging_value_enabled = KnownKey.newKey(false);
```

Comment:
instead of `modify_prefs_logging_value_enabled` how about `prefs_settings_value_logging_enabled`
and `prefs_settings_value_logging_delimeter`
for character limit `prefs_settings_value_logging_chars_limit`

Reply:
OK

Reply:
Done